Translation of 'how-to-contribute' and 'css libraries...' (#3)
gatsbyjs/gatsby-es: docs/contributing/community-contributions.md

---
title: Contribuciones de la Comunidad
---

> Una gran parte de lo que hace Gatsby grandioso son todos y cada uno de ustedes en la comunidad. Sus contribuciones enriquecen la experiencia de Gatsby y mejorarlo cada día.

Reviewer: ```suggestion
Una gran parte de lo que hace a Gatsby grandioso son todos y cada uno de ustedes en la comunidad. Sus contribuciones enriquecen la experiencia de Gatsby y lo hacen mejorar cada día.
```

Agradecemos todas sus contribuciones en la comunidad, y estaríamos encantados de amplificar su voz. Las contribuciones no se limitan al código, y pueden adoptar muchas formas.

> - Tu maravilloso websitio Gatsby.js!

Reviewer: ```suggestion
- Tu maravilloso sitio web Gatsby.js!
```

"Website" pasa a ser "sitio web" 🚀

> - entrantes de Gatsby

Reviewer: ```suggestion
- Starters de Gatsby
```

@horacioh Tenemos que hablar sobre la traducción de "starters" creo que se puede dejar sin traducir, o buscar una traducción mas fiel, no se que palabra puede ser, "iniciador" suena muy raro, no se que tan usado es en Español, en latinoamerica no lo usamos tanto, que tal en Europa? 🤔

Author: Revisé el diccionario y 'entrantes' es muy incorrecto. Pensé que es como la palabra inglés 'entrant'.

Y 'los principiantes' no es la misma de 'Starters'.

No se :) Por ahora, 'Starters'?

> - plugins de Gatsby

Reviewer: ```suggestion
- Plugins de Gatsby
```

> - Entradas de blog y estudios de caso

Reviewer: ```suggestion
- Entradas de blog y casos de estudio
```

> - Videos de conferencias

Reviewer 2: linea 13: Videos de charlas y conferencias

> - Materiales de los talleres

Reviewer: ```suggestion
- Materiales de talleres
```

> - Materiales de talleres

Reviewer 2: linea 14: - Material para talleres

> - Temas de GitHub y las solicitudes de extracción

Reviewer: ```suggestion
- Issues de GitHub y pull requests
```

@horacioh Otro caso, creo que "issues" y "pull requests" tampoco traducen bien, no se que opinión tengas acá.

> ...y cualquiera que se te ocurra. Nuestras muestras y documentos de contribución a bibliotecas son un lugar excelente para empezar:

Reviewer 2: linea 17: ...y cualquier cosa que se te ocurra. ...

> ...y cualquiera que se te ocurra. Nuestro escaparate y documentos de contribución de las bibliotecas están un lugar excelente para empezar.

Reviewer: ```suggestion
...y cualquiera que se te ocurra. Nuestras muestras y documentos de contribución a bibliotecas son un lugar excelente para empezar:
```

> - [Enviar al Escaparate del sitio](/contributing/site-showcase-submissions)

Reviewer: ```suggestion
- [Enviar a las muestras de sitios](/contributing/site-showcase-submissions)
```

> - [Enviar a la Biblioteca del Entrantes](/contributing/submit-to-starter-library/)

Reviewer: ```suggestion
- [Enviar a la biblioteca de Starters](/contributing/submit-to-starter-library/)
```

> - [Enviar a la Biblioteca del Plugins](/contributing/submit-to-plugin-library/)

Reviewer: ```suggestion
- [Enviar a la biblioteca de Plugins](/contributing/submit-to-plugin-library/)
```

> Examina el resto de nuestros documentaciones para contributors y contactarnos en [Twitter](https://twitter.com/gatsbyjs), [Discord](https://gatsby.dev/discord), o en un [Tema de GitHub](/contributing/how-to-file-an-issue/) si tienes alguna pregunta sobre la contribución!

Reviewer: ```suggestion
Examina el resto de nuestras documentaciones para contribuidores y contáctanos en [Twitter](https://twitter.com/gatsbyjs), [Discord](https://gatsby.dev/discord), o en un [Issue de GitHub](/contributing/how-to-file-an-issue/) si tienes alguna pregunta sobre contribuir!
```
